mindsdb_native/libs/phases/type_deductor/type_deductor.py

```diff
@@ -198,8 +202,27 @@ def get_column_data_type(self, data, full_data, col_name):
         else:
             curr_data_type, curr_data_subtype = None, None
 
+        # Check for Tags subtype
+        if curr_data_subtype != DATA_SUBTYPES.ARRAY:
+            lengths = []
+            unique_tokens = set()
+            can_be_tags = True
+            for item in data:
+                try:
+                    item_tags = [t.strip() for t in item.split(',')]
+                    lengths.append(len(item_tags))
+                    unique_tokens = unique_tokens.union(set(item_tags))
+                except AttributeError:
+                    can_be_tags = False
+                    break
+
+            # If more than 30% of the samples contain more than 1 category and there's more than 10 of them and they are shared between the various cells
+            if can_be_tags and np.mean(lengths) > 1.3 and len(unique_tokens) >= 6 and len(unique_tokens)/np.mean(lengths) < (len(data)/4):
```

Comment:
Is the final criterion 6 or 10 unique categories?

Reply:
6